### ISSUE TITLE
Series - Manual Import - Remapping Episodes w/ Unmonitored on Delete causes Episode to be Unmonitored.

### ISSUE THREAD

**Issue by bakerboy448** [Sonarr/Sonarr#4638]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Remapping a file from Episode 28 to Episode 28 and 29 - when unmonitor when deleted is enabled - results in episode 28 being unmonitored.  The logs note this was a non-history event.

### Expected Behavior

- Only history event deletes should trigger an episode to unmonitor.
- Alternatively: manual imports involving remapping existing files should not unmonitored any episodes.

### Steps To Reproduce

1.  Enable `Unmonitor Deleted Episodes`
2. Go to a Series
3. Manual Import
4. Map an existing file to a different or different grouping of episodes
5. Note that episode is unmonitored.

### Environment

```markdown
- OS: agnostic
- Sonarr: 3.0.6.1309
- Docker Install: agnostic
- Using Reverse Proxy: agnostic
- Browser: agnostic
```

### What branch are you running?

Develop

### Anything else?

Note same log file, but 1st is the entire log; 2nd is just the snippets after importing

- [Gist - Full Trace Logs](https://gist.github.com/bakerboy448/6ab0a4b202ad1ab93a57763b9442edff)
- [Gist - Snippets only Trace Logs](https://gist.github.com/bakerboy448/09bb0660a9ad9e70210632d67feb2ef5)

`2021-08-25 00:35:23.2|Trace|Http|Req: 77226 [PUT] /api/v3/episode/monitor` is me re-monitoring the episode

[Reproduction Logs](https://gist.github.com/bakerboy448/a7d7e36e8ffdd63f3a552c092a98d970) - Confirmed by repeating the steps to change to only map that file to E28 from E28 and E29.  Both E28 and E29 were unmonitored.  Refresh confirmed not Signalr/UI weirdness.

**Comment by bakerboy448** [comment]
- Seems a manual import from the series page also does not rename the file.  Is this intentional or an oversight?

**Comment by markus101** [comment]
> Seems a manual import from the series page also does not rename the file. Is this intentional or an oversight?

Intentional, it's not an import and rename, it's just importing the file, same as import from the series folder doesn't auto rename.
